chromium_src/chrome/browser/ui/views/tabs/tab_drag_controller.cc

```diff
@@ -107,7 +107,11 @@ void TabDragController::MoveAttached(const gfx::Point& point_in_screen,
   int index_of_last_item = attached_model->GetIndexOfWebContents(last_contents);
   if (index_of_last_item !=
       attached_model->GetIndexOfWebContents(last_contents)) {
-    last_move_attached_context_loc_ = point_in_screen.y();
+    gfx::Point point_in_attached_context =
```

Comment:
nit: why not simply `gfx::Point point_in_attached_context = point_in_screen;`?

Reply:
Yeah, good call. Thanks